client-data.md

```diff
@@ -1,21 +1,17 @@
 # Filecoin Client Data Processing
 
-Data that clients wish to store must be pre-processed before it can be sent to miners and have a storage deal made for it.
+Client must pre-process their data into a valid formats before sending them to miners, creating storage deals and generating mining proofs.
 
-This processing is comprised of two top-level pieces:
+There are two formats:
 
-1. **Transfer Encoding**
-2. **Storage Encoding**
+- **Transfer Format**
+- **Storage Format**
 
-Transfer Encoding is a pre-processing for the file to allow it to be transferred over the network more easily. Both for the initial transfer to the storage miner, and for later retrievals. This mechanism can be done in any way that is supported by the client and the miner (alternative strategies may be implemented).
+## Transfer Format
 
-Storage Encoding is a pre-processing for the file that allows the Filecoin storage market and proofs to reference the data. Filecoin operates on a special type of merkle tree generated with a function we call `RepHash`. The root hash of this merkle tree is called the `Piece Commitment` or `CommP` and is included in the deal when it is proposed to a miner.
+The Transfer Format is the format to transfer a file over the network. This format is used for the initial transfer (from clients to storage miners) and for later retrievals (from storage miners to the clients).
```

Comment:
(1) I quite preferred encoding since that is exactly what you describe but I imagine you changed to format because of collision with namings in the proofs or stg ?
(2) Is "Transfer Format" really a "proper name" ? simply have it like "transfer format" seems simpler (i.e. having it a proper name doesn't bring any benefit imo)